framework/src/play/Play.java

```diff
@@ -114,7 +114,7 @@ public boolean isProd() {
     /**
      * Main routes file
      */
-    public static VirtualFile routes;
+    public static List<VirtualFile> routes;
```

Comment:
It breaks backward compatibility.
I suggest leaving `VirtualFile routes` untouched and adding a new field `List<VirtualFile> internationalizedRoutes`. In this case you don't need field `multilangRouteFiles`.